src/Coduo/PHPHumanizer/Number/Spell.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Coduo\PHPHumanizer\Number;
+
+use Coduo\PHPHumanizer\Translator\Builder;
+
+// Credit to Karl Rixon: http://www.karlrixon.co.uk/writing/convert-numbers-to-words-with-php/
+class Spell
+{
+    /**
+     * @var \Symfony\Component\Translation\TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @var string
+     */
+    private $hyphen;
+
+    /**
+     * @var string
+     */
+    private $conjunction;
+
+    /**
+     * @var string
+     */
+    private $separator;
+
+    /**
+     * @var string
+     */
+    private $negative;
+
+    /**
+     * @var string
+     */
+    private $decimal;
+
+    /**
+     * @var array
+     */
+    private $map = array(
```

Comment:
Oh right. No problem. I thought you were favouring implicit scoping for the properties. I'll revert that commit.